opplast/upload.py

```diff
@@ -126,21 +135,18 @@ def upload(self, meta: dict) -> (bool, str):
                     "Did not set tags. Tags cannot be longer than 500 characters"
                 )
 
-        self.driver.find_element_by_id(NEXT_BUTTON).click()

```

Comment:
Why did you delete the 3rd "next" click? When I upload, I get 3 tabs. By only clicking next twice I would run into an exception, because it can't find the public button.
![image](https://user-images.githubusercontent.com/30203217/111871861-8da8aa80-898c-11eb-851d-ec2b6a1230fb.png)

Reply:
this is pretty weird since i only got 3 tabs 
![image](https://user-images.githubusercontent.com/36931468/111873024-9ac69900-898e-11eb-8949-5fb15210c6d1.png)

maybe we should wrap the thrid next button around try catch to solve this :)

Reply:
fixed it